docs/v3/codecs/sharding-indexed/v1.0.rst

```diff
@@ -25,12 +25,12 @@ is licensed under a `Creative Commons Attribution 3.0 Unported License
 Status of this document
 =======================
 
-.. rst-class:: draft
+ZEP0002 was accepted on November 1st, 2023 via https://github.com/zarr-developers/zarr-specs/issues/254.
 
 .. warning::
-    This document is a draft for review and subject to changes.
+    This document is subject to changes.
     It will become final when the `Zarr Enhancement Proposal (ZEP) 2 <https://zarr.dev/zeps/draft/ZEP0002.html>`_
-    is approved via the `ZEP process <https://zarr.dev/zeps/active/ZEP0000.html>`_.
+    is finalized via the `ZEP process <https://zarr.dev/zeps/active/ZEP0000.html>`_.
```

Comment:
Aren't there still a few more issues/prs though? :smile:

Reply:
Doesn't it say that it *will* become final when finalized?